buildSrc/src/main/kotlin/Config.kt

```diff
@@ -1,5 +1,5 @@
 object Config {
-    const val version = "8.0.2"
+    const val version = "9.0.0-SNAPSHOT"
```

Comment:
Is this `SNAPSHOT` supposed to be here?

Reply:
Yes, I forgot to add it in #2182 - it will be removed in the PR for release.